src/Components/Dashboard.js

```diff
@@ -22,9 +21,9 @@ const Dashboard = ({query}) => {
   const [user, setUser] = useState(['']);
 
   useEffect(() => {
-    Service.get('products').then(products => setProducts(products));
+    Service.get('products', "query="+ query).then(products => setProducts(products));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8502927)
```suggestion
    Service.get('products', "query="+ query).then((products) => setProducts(products));
```